examples/connectivity/plot_adhd_covariance.py

```diff
@@ -8,11 +8,26 @@
  
 plotted_subject = 0  # subject to plot
 
+import numpy as np
 
 import matplotlib.pyplot as plt
+
+from nilearn import plotting, image
 from nilearn.plotting import cm
 
 
+def plot_connectome(cov, atlas_maps, **kwargs):
```

Comment:
> The thing is that this function is used twice, one for the
> GroupSparseCovariance and one for the GraphLasso.

Yes, but the only thing that needs to be called twice is the call to
plot_connectome. The rest is common.

> Also there is already a function to plot the matrices in this example.

I don't like that either :).